requirements.txt

```diff
@@ -0,0 +1,35 @@
+argon2-cffi==20.1.0
+cffi==1.14.5
+dill==0.3.3
+fn==0.4.3
+funcy==1.15
+idna==2.10
+ipywidgets==7.6.3
+jedi==0.18.0
+matplotlib==3.3.2
+multiprocess==0.70.11.1
+nbconvert==6.0.7
+nbformat==5.1.3
+networkx==2.5
+notebook==6.3.0
+numba==0.51.2
+numpy==1.19.5
+pyrsistent==0.17.3
+packaging==20.9
+pandas==1.1.5
+parameterized==0.7.4
+parso==0.8.2
+plotly==4.10.0
+pox==0.2.9
+ppft==1.6.6.3
+py==1.10.0
+Pygments==2.8.1
+pytz==2021.1
+QtPy==1.9.0
+scikit-learn==0.23.2
+scipy==1.5.2
+seaborn==0.11.0
+six==1.15.0
+tabulate==0.8.7
+toml==0.10.2
+xarray==0.16.0
```

Comment:
E. needed for labs
MUST have `requirements.txt`
_At the very least_ the dependencies for executing your simulation locally